build/generate/featureMatrix.go

```diff
@@ -34,6 +34,7 @@ func generateFeatureMatrix() error {
 			{"CAA", "Provider can manage CAA records"},
 			{"PTR", "Provider supports adding PTR records for reverse lookup zones"},
 			{"NAPTR", "Provider can manage NAPTR records"},
+			{"SOA", "Provider can manage SOA records"},
```

Comment:
@tlimoncelli I did - was I supposed to add the generated 'docs/_includes/matrix.html' file to the PR?

Reply:
Ooops!  You certainly did!  My bad.